examples/complex.bs.js

```diff
@@ -0,0 +1,17 @@
+// Generated by BUCKLESCRIPT VERSION 3.1.5, PLEASE EDIT WITH CARE
```

Comment:
I don't think we need to commit javascript code of examples.

Reply:
You right, I remove them.